src/main/java/core/basesyntax/model/figure/Figure.java

```diff
@@ -1,5 +1,8 @@
 package core.basesyntax.model.figure;
 
+import javax.persistence.MappedSuperclass;
+
+@MappedSuperclass
```

Comment:
I assume if we mark this class as `MappedSuperClass` - we can create field `Id` in `Figure` class with corresponding strategy which will be inherited by all sub classes

Reply:
done